### ISSUE TITLE
Tons Of erros 

### ISSUE THREAD

**Issue by adhxtry** [codemation/aiopyql#3]
![](https://cdn.discordapp.com/attachments/839859783771488306/841950814369218570/unknown.png)

![](https://cdn.discordapp.com/attachments/839859783771488306/841951264425771028/unknown.png)
I just copied and pasted the sample code from docs, and these errors spammed.

### CODE:

```py
import asyncio
from aiopyql import data

b= ''

async def main():

    #sqlite connection
    db = await data.Database.create(...DATABASE info...)

    # create table
    await db.create_table(
        'keystore',
        [
            ('key', str, 'UNIQUE NOT NULL'),
            ('value', str)
        ],
        'key',
        cache_enabled=True
    )

    # insert
    await db.tables['keystore'].insert(
        key='foo',
        value={'bar': 30}
    )

    # update
    await db.tables['keystore'].update(
        value={'bar': 31},
        where={'key': 'foo'}
    )

    # print values
    b= await db.tables['keystore'].select(
        'key',
        where= {'key': 'foo'}
    )
    print(b)

print(b)
asyncio.run(main())
```

**Comment by codemation** [comment]
Thanks @AdhTri-001 for the report.

I was able to reproduce the same behavior which seems to have escaped tests because restart-ability was not covered in tests. 

## Problem
Essentially a table can be created correctly, but when loaded ( i.e the table already exists) at startup, postgres returns a table schema including VARYING(36) which is an alias for VARCHAR(36) so it is interpreted as column modifier by aiopyql. 

## Solution

- [x] Added a column modifier exclusion for VARYING, since there is a hard conversion via aiopyql [str -> VARCHAR(36) in postgres] if the column is a primary key, using VARYING is not expected. 
- [x] Increase testing coverage - add additional tests for restart-ability after aiopyql is restarted and loading existing tables.

**Comment by codemation** [comment]
@AdhTri-001 

The fix for the reported issues is available via the latest version of aiopyql 0.358
https://pypi.org/project/aiopyql/0.358/

To install - you can run the following command - twice to force an update via pip
```bash
pip install -U aiopyql
& 
pip install -U aiopyql

Installing collected packages: aiopyql
  Attempting uninstall: aiopyql
    Found existing installation: aiopyql 0.357
    Uninstalling aiopyql-0.357:
      Successfully uninstalled aiopyql-0.357
Successfully installed aiopyql-0.358
```
Try it out and let me know how it goes!

**Comment by adhxtry** [comment]
Wow Its working as it should now, no more spams of errors, YAY thanks @codemation U are awesome
